### PR TITLE
Feature plotting

### PR DESCRIPTION
**Pull request by clarkfitzg**
Opening to get feedback as work is done here. Don't merge yet.

### CODE REVIEW

File: xray/core/plotting.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+def _plot_dataarray(darray, *args, **kwargs):
+    """
+    Plot a DataArray
+    """
+    import matplotlib.pyplot as plt
+
```
Comment (shoyer) [inline xray/core/plotting.py]: Probably should raise an error if the data is not 1D

File: xray/core/plotting.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    xlabel = darray.indexes.keys()[0]
+    x = darray.indexes[xlabel].values
+    y = darray.values
+
+    # Probably should be using the lower level matplotlib API
+    plt.plot(x, y, *args, **kwargs)
+    ax = plt.gca()
```
Comment (shoyer) [inline xray/core/plotting.py]: It would be a good idea to make sure there is a way of passing in this argument.

File: xray/core/plotting.py (hunk 1 above)
Comment (jhamman) [inline xray/core/plotting.py]: For 1D plotting, would it be possible to use the `pandas` plot functionality, rather than rewrite it here?

Comment (clarkfitzg) [inline xray/core/plotting.py]: We have been considering that. `pandas` plotting does much more than we're planning to put here. The goal here is to use the metadata to make simple plots.

**Comment by clarkfitzg** [comment]
Moving development to this branch: https://github.com/xray/xray/tree/feature-plotting

**Pull request closed without merging**